fractal/parser.py

```diff
@@ -304,7 +305,11 @@
     help="New resource type",
 )
 task_edit_parser.add_argument(
-    "--default-args", help="Filename containing JSON encoded default arguments"
+    "--default-args-file",
+    help="Filename containing JSON encoded default arguments",
+)
+task_edit_parser.add_argument(
+    "--meta-file", help="Filename containing JSON encoded default arguments"
```

Comment:
FIXME: help message

Reply:
Something like "Path to JSON serialised file containing the task meta dictionary" would work